Services/WindowServer/WindowManager.cpp

```diff
@@ -163,6 +163,22 @@ Gfx::IntSize WindowManager::resolution() const
     return Screen::the().size();
 }
 
+void WindowManager::set_acceleration_factor(double factor)
+{
+    Screen::the().set_acceleration_factor(factor);
+    dbg() << "Saving acceleration factor: " << factor << " to config file at " << m_config->file_name();
```

Comment:
Please use `dbgln()` going forward, the streaming variants are being phased out :)

```suggestion
    dbgln("Saving acceleration factor {} to config file at {}", factor, m_config->file_name());
```

Reply:
fixed!